exist-core/src/main/java/org/exist/source/FileSource.java

```diff
@@ -103,7 +103,7 @@ public InputStream getInputStream() throws IOException {
     @Override
     public String getContent() throws IOException {
         checkEncoding();
-        return new String(Files.readAllBytes(path), encoding);
+        return Files.readString(path, encoding);
```

Comment:
Will this different method handle binary files as well?

Reply:
I do not fully understand your question, but the  new method does the same thing as the old invovation and was added with Java 11